src/main/scala/org/broadinstitute/dsde/agora/server/webservice/PerRequest.scala

```diff
@@ -62,34 +66,33 @@ trait PerRequest extends Actor with LazyLogging {
 
   override val supervisorStrategy =
     OneForOneStrategy(loggingEnabled = AgoraConfig.supervisorLogging) {
+      case e => {
+        handleException(e)
+        Stop
+      }
+    }
+
+  def handleException(t: Throwable) = t match {
```

Comment:
this exception handling is more in line with what rawls and orch do, with the difference that Agora rewrites its internal exceptions to different statuses all at once rather than having a `RequestCompleteWithErrorReport` structure